website/docs/d/cos_bucket.html.markdown

```diff
@@ -168,7 +168,7 @@ In addition to all argument reference list, you can access the following attribu
   - `rule_id` - (String)  Unique identifier for the rule. Expire rules allow you to set a specific time frame after which objects are deleted.
 - `hard_quota` - (String) Maximum bytes for the bucket.
 - `id` - (String) The ID of the bucket.
-- `key_protect` - (String) The CRN of the IBM Key Protect instance where a root key is already provisioned. 
```

Comment:
Are you planning to remove old argument?
In that case, Kindly add deprecation notice as well.